lib/helpers.php

```diff
@@ -123,35 +123,35 @@ function twofa_verify_token($secret, $token)
     return $otp->checkTotp(\Base32\Base32::decode($secret), $token, TWOFA_WINDOW);
 }
 
-// Stores a token in the blacklist and returns true if the token was already in the blacklist
-function twofa_token_blacklist($token)
+// Stores a token in the denylist and returns true if the token was already in the denylist
+function twofa_token_denylist($token)
 {
-    // Retrieve blacklist
-    $blacklist = get_site_option('2fa_token_blacklist');
-    if (!is_array($blacklist)) {
-        $blacklist = [];
+    // Retrieve denylist
+    $denylist = get_site_option('2fa_token_denylist');
```

Comment:
Do we mind this will lose the details of any previously denied tokens? Given the list is cleared after 5 minutes, I'm guessing it's not too big a deal, but might want to be sure.

Reply:
It's a good question. I think maybe it's worth backing these changes out, and then adding them in a separate PR, with the semvar / ChangeLog that you've suggested.